### DISCUSSION TITLE
Quoting settings

### DISCUSSION THREAD

**Discussion by ivanbuto** [macros4outlook/quotefixmacro#26] [Q&A]
In the readme, the following instructions are given to set up email in Outlook:

> Tools > Options > Preferences > E-mail Options… > On replies and forwards
> - Change the value “When replying to a message” to “Prefix each line of the original message”
> - Set “Prefix each line with” to “> ”
> - Change the value “When replying to a message” back to to “Include original message text”

Is this correct? If I change the quoting setting back to "include original message text", and then use FixedReply, I get an empty citation (no quoted text at all).

The problem is that when replying to HTML e-mails, which I use most of the time, I might want to keep the "include original message text" option. Is there any way to have this combination, or do I have to change the setting to prefix messages, which will also impact HTML messages?

Also do you know if there is any way to assign other icons to the custom FixedReply button other than the icons offered by Outlook?
![image](https://user-images.githubusercontent.com/34988265/156899680-9985b94f-9034-4c9a-8432-acb6cc942d99.png)

I would like to use the same icon as is displayed on the standard Reply button, but it's not available ones.

**Comment by Mo-Gul** [comment]
First to the second question about the icon, because this is short: Unfortunately I don't have a clue how to do that. If you eventually find a solution, please let me/us know!

To your first question/issue: Could you let us know which (exact) version of QuoteFixMacro you are using? Because this shouldn't happen. As you can see here

https://github.com/macros4outlook/quotefixmacro/blob/8ab94222fae5d8005ece88dc651793768e4989f8/QuoteFixMacro.bas#L760-L776

non-Plain-Text emails should be handled as if you clicked the (normal) Reply button (assuming `CONVERT_TO_PLAIN` is still the default `DEFAULT_CONVERT_TO_PLAIN` (which is `False`)).
